DEVELOPER_GUIDE.md

```diff
@@ -55,7 +55,32 @@ Bound addresses will then be logged to the terminal :
 
 ## Run OpenSearch
 
-- Start a seperate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`
+- Start a separate terminal and navigate to the directory that OpenSearch has been cloned to using `cd OpenSearch`.
+
+- Check if extensions directory exists in OpenSearch using `ls`.
+- If the directory does not exist, create it using `mkdir extensions`.
+- Navigate to the extensions folder using `cd extensions`.
+- Manually create a file titled `extensions.yml` within the extensions directory using an IDE or an in-line text editor.
+
+Sample extensions.yml file:
+
+```
+extensions:
+  - name: opensearch-sdk
+    uniqueId: opensearch-sdk-1
+    hostName: 'sdk_host'
+    hostAddress: '127.0.0.1'
+    port: '4532'
+    version: '1.0'
+    description: Extension for the Opensearch SDK Repo
```

Comment:
I get errors when Extension is attempt to be created from this yml file, eg:

`OpenSearchException[failed to bind service]; nested: IOException[Could not read from extensions.yml]; nested: UnrecognizedPropertyException[Unrecognized field "description" (class org.opensearch.extensions.ExtensionsSettings$Extension), not marked as ignorable (6 known properties: "port", "hostName", "version", "name", "uniqueId", "hostAddress"])`

   `at [Source: (sun.nio.ch.ChannelInputStream); line: 8, column: 57] (through reference chain: org.opensearch.extensions.ExtensionsSettings["extensions"]->java.util.HashSet[0]->org.opensearch.extensions.ExtensionsSettings$Extension["description"])];
`

Is this a bug in the documentation or something that needs to be fixed in the product?